app/src/routes/routesMap.js

```diff
@@ -123,7 +113,7 @@ const routesMap = {
     },
   },
   [PROJECT_DETAILS_PAGE]: {
-    path: `/administrate/projects/:projectId/:projectSection(${SETTINGS}|${MEMBERS}|${EVENTS})?/:settingsTab(${GENERAL}|${NOTIFICATIONS}|${INTEGRATIONS}|${DEFECT}|${ANALYSIS}|${PATTERN_ANALYSIS}|${DEMO_DATA})?`,
+    path: `/administrate/projects/:projectId/:projectSection(${SETTINGS}|${MEMBERS}|${EVENTS})?/:settingsTab?`,
```

Comment:
So how will unknown tabs be handled? (see the screenshot below for version 5.1)

![image](https://user-images.githubusercontent.com/31861805/76069178-842bea80-5fa3-11ea-9339-97c1b3434cc4.png)

Reply:
I can show the first tab of the list. Is it ok for now?

Reply:
(Now it throws error because no config for this unknow tab found)

Reply:
> I can show the first tab of the list. Is it ok for now?

Yes, it is.